.github/workflows/run_tests.yml

```diff
@@ -45,15 +45,15 @@ jobs:
     strategy:
       matrix:
         go_versions: [ '1.22', '1.23' ]
-        os: [ubuntu-22.04, windows-2022, macos-14]
+        os: [ubuntu-latest, windows-2022, macos-14]
```

Comment:
We may use `windows-latest` and `macos-latest` as far, ref. https://github.com/actions/runner-images?tab=readme-ov-file#available-images. Do we need it?

Reply:
At some point this wasn't really possible since latest macs were amd64 and we wanted arm64. There was some problem with windows as well. I'd leave these for now, but im future we can switch.